data/json/mapgen/lab/lab_modular/lab_nests_modular/lab_shared_traps.json

```diff
@@ -36,5 +36,105 @@
     "update_mapgen_id": "concrete_wall_o_lab_res_8_NE",
     "method": "json",
     "object": { "place_terrain": [ { "ter": "t_thconc_floor", "x": 12, "y": 15 }, { "ter": "t_thconc_floor", "x": 12, "y": 16 } ] }
+  },
+  {
+    "type": "trap",
+    "id": "tr_light_on_res_8_NW",
+    "name": "overhead light",
+    "color": "white",
+    "symbol": ".",
+    "visibility": 99,
+    "avoidance": 99,
+    "difficulty": 99,
+    "action": "map_regen",
+    "map_regen": "light_on_res_8_NW",
+    "benign": false
```

Comment:
Should these be marked as benign? Bots, etc are unlikely to be trying to avoid them.

Reply:
Sigh... turns out 'benign' also means that the trigger turns off. `monster_no_avoid` as a boolean to add?